src/components/ChecBoxList/CheckBoxList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { skills } from '../../mock-data/skills.json';
+import CheckBoxItem from '../CheckBoxItem/CheckBoxItem';
+import './CheckBoxList.scss';
+
+function CheckBoxList({ checkedState, onChange }) {
+  const handleCheckedState = ({ target: { name, checked } }) => {
+    const result = checked
+      ? checkedState.concat(name)
+      : checkedState.filter((item) => item !== name);
+    onChange(result, name, checked);
+  };
+
+  const checkboxes = skills.map((item) => (
+    <CheckBoxItem
+      name={ item.name }
+      key={ item.id }
+      id={ item.id }
+      isSelected={ checkedState.includes(item.name) }
+      onChange={ handleCheckedState }
+    />
+  ));
+
+  return (
```

Comment:
Unless there is a reason to have this `<div>`, just remove it and return the result from the skills.map()